src/prof.c

```diff
@@ -310,6 +310,7 @@ prof_leave(tsd_t *tsd, prof_tdata_t *tdata) {
 }
 
 #ifdef JEMALLOC_PROF_LIBUNWIND
+JEMALLOC_ALIGNED(CACHELINE)
```

Comment:
Why do we want to cacheline-align this function?  Is there something strange going on with libunwind that this works around?